turbinia/workers/workers_test.py

```diff
@@ -204,7 +209,8 @@ def testTurbiniaTaskValidateResultBadResult(self, _):
     self.assertNotEqual(new_result.error, {})
 
   @mock.patch('turbinia.workers.evidence_decode')
-  def testTurbiniaTaskEvidenceValidationFailure(self, evidence_decode_mock):
+  @mock.patch('turbinia.workers.evidence_decode')
```

Comment:
What's this for?  Looks like you're mocking the same thing twice?

Reply:
Fixed.